goc/etc/cron.d/cvmfs

```diff
@@ -1,7 +1,7 @@
-* * * * * root test -f /var/lib/oasis/installed && cvmfs_server snapshot -ais
-* * * * * root test -f /var/lib/oasis/installed && sleep 15 && cvmfs_server snapshot -ais
-* * * * * root test -f /var/lib/oasis/installed && sleep 30 && cvmfs_server snapshot -ais
-* * * * * root test -f /var/lib/oasis/installed && sleep 45 && cvmfs_server snapshot -ais
+* * * * * root test -f /var/lib/oasis/installed && sem --id snapshot -j +0 --st -1 cvmfs_server snapshot -ais 2>/dev/null || true
+* * * * * root test -f /var/lib/oasis/installed && sleep 15 && sem --id snapshot -j +0 --st -1 cvmfs_server snapshot -ais 2>/dev/null || true
+* * * * * root test -f /var/lib/oasis/installed && sleep 30 && sem --id snapshot -j +0 --st -1 cvmfs_server snapshot -ais 2>/dev/null || true
+* * * * * root test -f /var/lib/oasis/installed && sleep 45 && sem --id snapshot -j +0 --st -1 cvmfs_server snapshot -ais 2>/dev/null || true
```

Comment:
Presumably the `2>/dev/null` is to discard `sem` errors when as many snapshot jobs as cores are already running.  But (in case it was not your intention), note that you are now also discarding any stderr from the cvmfs server snapshot command also.

Reply:
It's ok, the snapshot -a option sends all the output including errors to log files.